cmd/process-agent/main.go

```diff
@@ -29,7 +29,8 @@ func main() {
 	rootCmd.PersistentFlags().StringVarP(&opts.pidfilePath, "pid", "p", "", "Path to set pidfile for process")
 	rootCmd.PersistentFlags().BoolVarP(&opts.info, "info", "i", false, "Show info about running process agent and exit")
 	rootCmd.PersistentFlags().BoolVarP(&opts.version, "version", "v", false, "Print the version and exit")
-	rootCmd.PersistentFlags().StringVar(&opts.check, "check", "", "Run a specific check and print the results. Choose from: process, connections, realtime")
+	rootCmd.PersistentFlags().StringVar(&opts.check, "check", "",
+		"Run a specific check and print the results. Choose from: process, connections, realtime, process_discovery")
```

Comment:
nit: `realtime` is not a check but we can fix in a separate PR

Reply:
It was always that way :(